ApiBundle/Transformer/NodeTransformer.php

```diff
@@ -127,6 +127,7 @@ protected function addMainAttributes(FacadeInterface $facade, NodeInterface $nod
             $facade->addRight('can_read', $this->authorizationChecker->isGranted(ContributionActionInterface::READ, $node));
             $facade->addRight('can_edit_data', $this->businessRulesManager->isGranted(BusinessActionInterface::EDIT, $node) && $this->authorizationChecker->isGranted(ContributionActionInterface::EDIT, $node));
             $facade->addRight('can_edit', $this->authorizationChecker->isGranted(ContributionActionInterface::EDIT, $node));
+            $facade->addRight('can_publish_node', $this->businessRulesManager->isGranted(NodeStrategy::CHANGE_TO_PUBLISH_STATUS, $node));
```

Comment:
@itkg-nanne : I think you should add this right in a GroupContext to avoid its check in all context, even if it's not necessary